sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS `quest_offer_reward_locale`;
+CREATE TABLE `quest_offer_reward_locale` (
+  `ID` int(10) unsigned NOT NULL DEFAULT '0',
+  `locale` varchar(4) NOT NULL,
+  `RewardText` text,
+  `VerifiedBuild` smallint(6) NOT NULL DEFAULT '0',
+  PRIMARY KEY (`ID`,`locale`)
+)

```

Comment:
There's no such table on the master branch, only on 3.3.5.
On 3.3.5 it looks grouped together in one table while in master if there is a table with enUS Text, there is (mostly) a second table *_locales with text locales.